sklearn/metrics/_classification.py

```diff
@@ -257,6 +261,14 @@ def confusion_matrix(y_true, y_pred, *, labels=None, sample_weight=None,
         samples with true label being i-th class
         and predicted label being j-th class.
 
+    Or
```

Comment:
This isn't valid numpydoc. The types need to be mentioned all on the first line.

Reply:
1. Changed parameter name from pprint --> as_dict()
2. Changed the Testing Function in test_classification.py
3. _Changed the Docstring for the function, added explaination of Series usage_

Not sure about the numpydoc please review.
- Should I mention it as --> tuple[ndarry, dict['true_class','pred_class']] ?